app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cd, old_file, new_file = command.split()
+    except ValueError:
+        raise ValueError("Write a command correctly. "
+                         "Example: cp file1.txt file2.txt")
```

Comment:
We have some condition in checklist: "Make sure you check that the `cp` input command is correct.". In your case you are raising exception when command is int for example =) Its good that you mentioned correct format but we can type any command here, like "mv file1.txt file2.txt" and it will work with this code.Correct me if I am wrong.